radixdlt-core/radixdlt/src/test/java/com/radixdlt/api/controller/HealthControllerTest.java

```diff
@@ -105,17 +117,34 @@ public void healthStatusIsReturned() {
 		when(exchange.getResponseHeaders()).thenReturn(new HeaderMap());
 		when(exchange.getResponseSender()).thenReturn(sender);
 		when(networkInfoService.nodeStatus()).thenReturn(BOOTING, SYNCING, UP, STALLED);
+		when(forkVoteStatusService.forkVoteStatus()).thenReturn(VOTE_REQUIRED, NO_ACTION_NEEDED, VOTE_REQUIRED, NO_ACTION_NEEDED);
```

Comment:
Let's not continue this pattern of unit tests which just set mocks and then test strings.